src/main/java/pl/fracz/mcr/source/Line.java

```diff
@@ -0,0 +1,117 @@
+package pl.fracz.mcr.source;
+
+/*
+ 2013-10-23, fracz, first implementation
+ 2013-10-30, fracz, added syntax highlighting
+ 2014-02-26, fracz, added ability to add voice comment
+ */
```

Comment:
to nie sa potrzebne w komentarzu informacje, powinny to być np. nazwy commitów raczej (first implementation, added syntax highling itd.)